jicofo/src/main/java/org/jitsi/jicofo/conference/JitsiMeetConferenceImpl.java

```diff
@@ -2055,11 +2055,11 @@ public void bridgeSelectionFailed()
         {
             ChatRoom chatRoom = getChatRoom();
             if (chatRoom != null
-                    && !chatRoom.containsPresenceExtension(
-                    BridgeNotAvailablePacketExt.ELEMENT,
-                    BridgeNotAvailablePacketExt.NAMESPACE))
+                    && chatRoom.getPresenceExtensions().stream().noneMatch(e ->
```

Comment:
Why did you remove containsPresenceExtension?

Reply:
To simplify the API.

Reply:
Well, I prefer the complexity to be either in the underlying implementation or the code that uses the interface, not the interface itself.

I changed it again. Now there's some extra complexity in ChatRoomImpl, but I think everything else is simpler.